pkgs/by-name/au/autocorrect/package.nix

```
@@ -4,24 +4,19 @@
   fetchFromGitHub,
 }:
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage (finalAttrs: {
   pname = "autocorrect";
-  version = "2.14.0";
+  version = "2.14.1";
 
   src = fetchFromGitHub {
     owner = "huacnlee";
     repo = "autocorrect";
-    rev = "v${version}";
-    sha256 = "sha256-Tqg0awxRtzqAVTTVrmN0RDTQvYJllejx8V94jTreZyI=";
+    rev = "5af1bc295d48b0fd04f7dbc35ea99d479f682e78";
```

A tag is available in the repo, so we should use it. :smile: 
```suggestion
    tag = "v${finalAttrs.version}";
```

---

Ah, I see. Then it's fine. Thank you for taking over the maintainership. :+1: